cloud/kubernetes/helm/values.yaml

```diff
@@ -5,103 +5,84 @@
 #Cloud Providers: minikube, PKS, Azure, AWS-EKS, Openshift
 
 #Required parameter to be passed for all cloud providers
-cloudProvider: minikube
-
-#Required parameter to be passed for cache, sharedNothing, sharedAll
-omType: inmemory
-
-#Required parameter to be passed for sharedNothing, sharedAll
-persistentType: None
+cpType: minikube
+#-----------------------------------------
+cmType: AS2                     #cmType can be unclustered,AS2,FTL
+omType: cache                   #omType can be inmemory,store,cache
+bsType: sharedNothing         #BackingStore can be None sharedNothing, store
+storeType: None                 #storeType can be RDBMS, AS4, Cassandra
```

Comment:
@LakshmiMekala valid values for storeType would be - None / Oracle / SQL Server / DB2 / MySQL / PostgreSQL / ActiveSpaces 4x (AS4) / Apache Cassandra

Reply:
All the RDBMS related values ( Oracle / SQL Server / DB2 / MySQL / PostgreSQL) will be considered as RDBMS.